mlflow_go/lib.py

```diff
@@ -16,6 +17,38 @@ def _get_lib_name() -> str:
     return "libmlflow-go" + ext
 
 
+def get_goarch():
```

Comment:
This reminds me of 90s-era browser detection code. Surely something abstracts and maintains this for us as a library, no?